lib/live_debugger_refactor/app/debugger/components_tree/web/components_tree_live.ex

```diff
@@ -162,9 +162,18 @@ defmodule LiveDebuggerRefactor.App.Debugger.ComponentsTree.Web.ComponentsTreeLiv
 
   defp highlight_element(
          %{assigns: %{highlight?: true, lv_process: %{pid: pid}}} = socket,
-         %{"search-attribute" => attr, "search-value" => val}
+         %{"search-attribute" => attr, "search-value" => val} = params
```

Comment:
Why don't you just do it like this? or use `params` for everything
```suggestion
         %{
           "search-attribute" => attr,
           "search-value" => val,
           "module" => module,
           "type" => type,
           "id" => id
         }

```